index.js

```diff
@@ -23,6 +22,7 @@ module.exports = function (opt) {
 
     var options = merge({
       bare: false,
+      coffee: require('coffeescript'),
```

Comment:
That's embarrassing... and explains the test failures. Good catch. I was using it to test coffeescript6.